configure.ac

```diff
@@ -4888,6 +4888,7 @@ if test "x$want_ech" != "xno"; then
   ECH_ENABLED=0
   ECH_ENABLED_OPENSSL=0
   ECH_ENABLED_WOLFSSL=0
+  ECH_ENABLED_RUSTLS=0
```

Comment:
When I was smoke-testing CI on my fork before opening this PR I noticed the [AM clang-tidy](https://github.com/cpu/curl/actions/runs/14065192995/job/39385760662) and [AM wolfssl-openssl-extra-valgrind](https://github.com/cpu/curl/actions/runs/14065192995/job/39385760490#logs) runs failed:

```
checking for wolfSSL_CTX_GenerateEchConfig... no
configure: error: --enable-ech ignored: No ECH support found
```

I wouldn't expect my diff here to break `wolfssl`'s support, but would appreciate if someone better at GNU autotools could take a peek.

Reply:
> I wouldn't expect my diff here to break wolfssl's support

Huh, well, they passed for the PR run so i'm going to chalk this up to a flake until proven otherwise.